scss/_breadcrumb.scss

```diff
@@ -1,6 +1,6 @@
 .breadcrumb {
   padding: $breadcrumb-padding-y $breadcrumb-padding-x;
-  margin-bottom: 1rem;
```

Comment:
Leave it as `margin-bottom`—there's no need to reset a non-existant `margin-top`, `-right`, and `-left` if we have no intention of ever applying those values (which I don't :p).